circuits/cpp/src/aztec3/circuits/abis/c_bind.cpp

```diff
@@ -153,7 +154,7 @@ WASM_EXPORT void abis__compute_function_selector(char const* func_sig_cstr, uint
     // hash the function signature using keccak256
     auto keccak_hash = ethash_keccak256(reinterpret_cast<uint8_t const*>(func_sig_cstr), strlen(func_sig_cstr));
     // get a pointer to the start of the hash bytes
-    uint8_t const* hash_bytes = reinterpret_cast<uint8_t const*>(&keccak_hash.word64s[0]);
+    auto const* hash_bytes = reinterpret_cast<uint8_t const*>(&keccak_hash.word64s[0]);
```

Comment:
Not sure I like that this is being changed from `uint8_t` to `auto`, but I can live with it.

Reply:
Consistent approach throughout, which I got used to, ove the 160 files :D

Reply:
It seems to only switch to `auto` when the type is obvious. And that is what I say to do in our standard.